libs/langchain/ragstack_langchain/colbert/colbert_retriever.py

```diff
@@ -37,9 +37,9 @@ def __init__(
         **kwargs: Any,
     ):
         super().__init__(retriever=retriever, k=k, **kwargs)
-        self._retriever = retriever
-        self._k = k
-        self._query_maxlen = query_maxlen
+        self.retriever = retriever
+        self.k = k
+        self.query_maxlen = query_maxlen
```

Comment:
fixes a bug instantiating the langchain colbert retriever from an existing vector store.